api/src/main/proto/spine/protodata/source.proto

```diff
@@ -70,3 +70,16 @@ message ProtobufSourceFile {
     //
     map<string, Service> service = 5;
 }
+
+// A Protobuf definitions file which is included into the module as a dependency.
+//
+// Unlike `ProtobufSourceFile`, `ProtobufDependencyFile` should not result in any code generation.
+// Rather, it serves an informative purpose.
+//
+message ProtobufDependencyFile {
```

Comment:
I think the suffix `File` in this type and in `ProtobufSourceFile` makes it a bit harder to understand the difference. More, we have the `file` property under `ProtobufSourceFile` and `content` under `ProtobufDependencyFile`, which makes it a bit more confusing. In fact, the dependency is on the file. We just don't generate the code after it.